### PR TITLE
CODENVY-88 Changed tomcat JVM defaults

### PR DESCRIPTION
**Pull request by skabashnyuk**
Remove unused in Java 8 properties
Used simple entropy source. See more https://wiki.apache.org/tomcat/HowTo/FasterStartUp

Need for https://github.com/eclipse/che/issues/498

### CODE REVIEW

**Comment by garagatyi** [comment]
Why you want to set unsecure random generation by defaut without providing posibility to enable secure mode easily?

**Comment by TylerJewell** [comment]
Won't this call never be called?

In che.sh, there is this code:
```
  ### Cannot add this in setenv.sh.
  ### We do the port mapping here, and this gets inserted into server.xml when tomcat boots
  export JAVA_OPTS="${JAVA_OPTS} -Dport.http=${CHE_PORT}"
```

So there is always a JAVA_OPTS.  And your code only adds in the defaults if a JAVA_OPTS doesn't exist. So shouldn't it be added into che.sh?  And we could then add a --blocking-entropy flag that will leave off the -D option.

File: che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat [hunk 1]
```diff
@@ -17,7 +17,7 @@ if "%CHE_LOCAL_CONF_DIR%"==""   (
    exit
 )
 
-if "%JAVA_OPTS%"=="" (set JAVA_OPTS=-Xms256m -Xmx2048m -XX:MaxPermSize=256m -server)
```
Comment (garagatyi) [inline che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat]: Can you explain why you removed `-server` option

**Comment by garagatyi** [comment]
In https://github.com/eclipse/che/pull/528 `-Dche.logs.dir=${CHE_LOGS_DIR}"` was in CATALINA_OPTS. Here it is used in JAVA_OPTS. Which way is correct?

**Comment by garagatyi** [comment]
I agree with @TylerJewell - review logic around usage of JAVA_OPTS. 
I think it is OK to set non-blocking mode as default if we add option to switch to blocking mode. @TylerJewell WDYT?

**Comment by TylerJewell** [comment]
@garagatyi This is my preference, but @skabashnyuk felt that it was important to let developers to override JAVA_OPTS.  But I think we can let users provide their own JAVA_OPTS, but we just have an additional parameter inside of Che.   So I think we revert this and let me work up an improvement to the che.sh script that adds all of these parameters.  And that we remove it out of the setenv.sh.

File: che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat (hunk 1 above)
Comment (skabashnyuk) [inline che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat]: According to docs

>Turns Java VM features specific to server applications, such as sofisticated JIT compiler. Though this option is implicitely enabled for x64 virtual machines, it still makes sence to use it as according to documentation behaviour maybe changed in the future.

This parameter was come from the old ages there all our tomcats running in 32-bit VM on server. Now I think this is no longer needed. And I want to let JVM decide the best mode.

File: che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat [hunk 2]
```diff
@@ -17,7 +17,7 @@ if "%CHE_LOCAL_CONF_DIR%"==""   (
...
+if "%JAVA_OPTS%"=="" (set JAVA_OPTS=-Xms256m -Xmx2048m -Djava.security.egd=file:/dev/./urandom)
```
Comment (benoitf) [inline che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat]: are you sure on windows you've /dev/urandom ?

File: che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat (hunk 1 above)
Comment (garagatyi) [inline che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat]: ok

File: che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat (hunk 2 above)
Comment (skabashnyuk) [inline che-tomcat8-slf4j-logback/src/assembly/bin/setenv.bat]: No.

**Comment by riuvshin** [comment]
OK

**Pull request merged**